dev_guide/creating_crd_objects.adoc

```diff
@@ -0,0 +1,19 @@
+[[dev-guide-custom-resources]]
+= Creating an object from a custom resource definition
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:context: dev-guide-crd
```

Comment:
Dev guide probably needs a snippet that says "let us assume the cluster admin has set up the following CRD for you in advance and has given you read and write access to it."